configurations/morcore2esuites/Configuration_TaakApiInt.xml

```diff
@@ -14,11 +14,13 @@
             </Exits>
 
         <SenderPipe name="Send2MorCore">
-            <HttpSender name="GetNotification" methodType="GET" url="/melding/{uuid}"/>
+            <HttpSender name="GetNotification" methodType="GET" url="${MorCore.taak.API.root-url}/melding/${meldingUuid}"/>
+                <Param name="meldingUuid" sessionKey="originalMessage" xpathExpression="//meldingUuid"/>
+                <Param name="url" value="${MorCore.taak.API.root-url}"/>
```

Comment:
You want the url only as a param, not both param and attribute. Also, if you are using variables that are only know during runtime like ${meldingUuid}, you can never pass the url by attribute. The framework always resolves attributes at startup. So referencing DeploymentSpecifics properties for example you could use in attributes, because they are known at startup, but you can't reference any runtime variables. Use param's for that instead.

It should be come:
<HttpSender name="GetNotification" methodType="GET" />
                <Param name="meldingUuid" sessionKey="originalMessage" xpathExpression="//meldingUuid"/>
                <Param name="url" value="${MorCore.taak.API.root-url}/melding/${meldingUuid}" />